src/Sulu/Bundle/MediaBundle/Tests/Application/Kernel.php

```diff
@@ -13,10 +15,22 @@
 
 use Sulu\Bundle\TestBundle\Kernel\SuluTestKernel;
 use Sulu\Component\HttpKernel\SuluKernel;
+use Symfony\Bundle\SecurityBundle\SecurityBundle;
 use Symfony\Component\Config\Loader\LoaderInterface;
 
 class Kernel extends SuluTestKernel
 {
+    public function registerBundles(): iterable
+    {
+        $bundles = parent::registerBundles();
+
+        if (SuluTestKernel::CONTEXT_WEBSITE === $this->getContext()) {
```

Comment:
This enables the `SecurityBundle` for all functional test cases of the `MediaBundle` that run in the website context.

Reply:
Updates this to create a new `WebsiteSecured` Testcase